README.md

```diff
@@ -3,6 +3,9 @@
 Source code for the CAV workshop/challenge.
 
 Here you can see the final code that you guys will be able to make by the end of the workshop. 
```

Comment:
Use two spaces or an extra new line to separate paragraphs. It'll currently look like this:
![image](https://user-images.githubusercontent.com/10410579/47688584-7c448500-dbbc-11e8-8503-cf385ca863ca.png)